code-challenges.test.js

```diff
@@ -67,3 +196,24 @@ const vowelTester3 = "challenges"
 
 
 // b) Create the function that makes the test pass.
+        //Create function called whereVowel that takes in a string as an argument for the parameter
+        const whereVowel = (string) => {
+            //turn the string entered as an argument into an array with each character occupying its own index
+            let array = string.split("")
+            //since .findIndex() needs a function to call upon, create a function that individually compares each value to each vowel
+            function checkVal(i) {
+                return i === "a" || i === "e" || i === "i" || i === "o" || i === "u"
+            }
+            return (array.findIndex(checkVal)) //.findIndex() executes my function for each array element and returns the index of the first element that passes the checkVal test
+       
+        }    
+
+//I would refactor, but this particular challenge took about 6 hours of my life. I originally tried to figure it out with written-out loops and other things on the syllabus, but I just wasnt writing the proper syntax or something.
```

Comment:
nested functions... like their order is somehow higher...

Reply:
Still great work! .filter with value and index being used are your friends here.